### PR TITLE
Migrate from AppVeyor to Github Actions #1836

### LINKED ISSUE #1836: Migrate from AppVeyor to Github actions
It would be great if we could get out of appveyor.

### PR DESCRIPTION
**Pull request by venky8951**
closes #1836

### CODE REVIEW

**Comment by kblok** [comment]
hmmm I can't see how I can approve these new actions

File: .github/workflows/demo.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: Demo Project
+
+on:
+  push:
+  pull_request:
+    branches: [ main ]
```
Comment (kblok) [inline .github/workflows/demo.yml:6]: We don't have a main branch yet :)

Comment (venky8951) [reply .github/workflows/demo.yml:6]: Hi, When we set branches to main it takes the default branch which is master, we can see Actions run here -> https://github.com/venky8951/puppeteer-sharp/actions

**Comment by kblok** [comment]
Thank you @venky8951!
I've made some edits. We do need to point to `master`.
I will need some follow-up PR but this is a great first step!

**Pull request merged**